javaparser-core/src/main/java/com/github/javaparser/javadoc/description/JavadocInlineTag.java

```diff
@@ -41,7 +41,11 @@ public static JavadocDescriptionElement fromText(String text) {
         text = text.substring(2, text.length() - 1);
         String tagName = nextWord(text);
         Type type = Type.fromName(tagName);
-        String content = text.substring(tagName.length());
+        int contentStart = tagName.length();
+        while (contentStart < text.length() && text.charAt(contentStart) == ' ') {
```

Comment:
Should we use trim? Should we skip any kind of whitespace? What about new lines?